The pin-by-pin button in the component editor was a push button, not 
a check button. It therefore worked as a toggle, as intended, but
lacked visual feedback.

- Werner

---

Index: kicad/eeschema/tool_lib.cpp
===================================================================
--- kicad.orig/eeschema/tool_lib.cpp	2009-05-20 22:40:19.000000000 -0300
+++ kicad/eeschema/tool_lib.cpp	2009-05-20 22:40:42.000000000 -0300
@@ -214,7 +214,8 @@
         m_HToolBar->AddSeparator();
         m_HToolBar->AddTool( ID_LIBEDIT_EDIT_PIN_BY_PIN, wxEmptyString,
                             wxBitmap( pin2pin_xpm ),
-                            _( "Edit pins part per part (Carefully use!)" ) );
+                            _( "Edit pins part per part (Carefully use!)" ),
+			    wxITEM_CHECK );
         m_HToolBar->ToggleTool( ID_LIBEDIT_EDIT_PIN_BY_PIN, g_EditPinByPinIsOn );
 
         // after adding the buttons to the toolbar, must call Realize() to reflect the changes
